oss-internship-2020/jsonnet/tests/jsonnet_tests_utils.cc

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "jsonnet_tests.h"
+
+// Prepare what is needed to perform a test.
+void JsonnetTestHelper::TestSetUp() {
+  // Get paths to where input and output is stored.
+  char buffer[256];
+  int error = readlink("/proc/self/exe", buffer, 256);
+  std::filesystem::path binary_path = dirname(buffer);
+  std::filesystem::path input_path =
+      binary_path / "tests_input" / "dummy_input";
+  std::filesystem::path output_path =
+      binary_path / "tests_output" / "dummy_output";
```

Comment:
Please use `sandbox2::file::JoinPath()` instead. `std::filesystem` is not well supported everywhere, yet.
See https://github.com/google/sandboxed-api/blob/master/sandboxed_api/sandbox2/util/path.h#L35